app/poros/movie.rb

```diff
@@ -2,25 +2,25 @@ class Movie
   attr_reader :rating, :runtime, :year_released, :language, :title, :poster_path, :genres
 
   def initialize(attributes)
-    @rating = rating(attributes)
+    @rating = find_rating(attributes)
```

Comment:
Thanks for renaming these methods, I now understand why I was getting the errors since we named the methods the same as our instance variables in our `attr_reader`.